GitUI/BuildServerIntegration/BuildServerWatcher.cs

```diff
@@ -93,6 +93,7 @@ public void CancelBuildStatusFetchOperation()
             }
         }
 
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Usage", "CA2202:Do not dispose objects multiple times")]
```

Comment:
@sharwell Technically you are right and MemoryStream disposed twice. How do you suggest to fix it? If we omit using on either MemoryStream or StreamReader it would be even more akward.

And here in the comments Jon Skeet agrees with me http://stackoverflow.com/questions/1065168/does-disposing-streamreader-close-the-stream